github/resource_github_team_repository.go

```diff
@@ -33,10 +33,9 @@ func resourceGithubTeamRepository() *schema.Resource {
 				ForceNew: true,
 			},
 			"permission": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				Default:      "pull",
-				ValidateFunc: validateValueFunc([]string{"pull", "triage", "push", "maintain", "admin"}),
```

Comment:
Is the reason for removing the validation to allow all custom role names?

Reply:
Yeah, that's it. GitHub itself would handle whether the role exists going forward

Reply:
Is there a way to pull all possible roles from GitHub and then validate against that?